business_requirement/__openerp__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © 2015
```

Comment:
for all copyrights:
© 2015 Elico Corp, contributor: Eric Caudal, Alex Duan, Xie XiaoPeng